### ISSUE TITLE
Using TextareaField with ref

### ISSUE THREAD

**Issue by MercuryVladaBelopolskaya** [segmentio/evergreen#1014]
Hey.
I want to use TextareaField inside a functional component wrapped in forwardRef.
When I use the ref attribute with the Textarea component, everything works fine, but if I want to add the ref attribute in the TextareaField, I get an error in the console: 

_Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?
Check the render method of `TextareaInput`._

My code: 
```
const TextareaInput = forwardRef((props, ref) => (
  <TextareaField
    label={props.label}
    name={props.name}
    isInvalid={props.isInvalid}
    validationMessage={props.invalidText}
    defaultValue={props.defaultValue}
    ref={ref}
  />
));
```

How can I use ref inside TextareaField ?

**Comment by akleiner2** [comment]
@MercuryVladaBelopolskaya what version of Evergreen are you using? We haven't cut a release _just_ yet with https://github.com/segmentio/evergreen/pull/987, which forwards the ref to the underlying `<Textarea />`, but be on the lookout for that some time today :)

**Comment by MercuryVladaBelopolskaya** [comment]
@akleiner2 Oh cool! I'll look forward to it :)

**Comment by akleiner2** [comment]
@MercuryVladaBelopolskaya This should be fixed in `5.1.0`, do you mind pulling in the latest version and letting us know if this works? Thanks!

**Comment by MercuryVladaBelopolskaya** [comment]
@akleiner2  Everything is working. Thank you so much :)
